sdk/servicebus/azure-messaging-servicebus-track1-perf/src/main/java/com/microsoft/azure/servicebus/perf/SendBatchTest.java

```diff
@@ -0,0 +1,50 @@
+package com.microsoft.azure.servicebus.perf;
+
+import com.azure.perf.test.core.TestDataCreationHelper;
+import com.microsoft.azure.servicebus.IMessage;
+import com.microsoft.azure.servicebus.Message;
+import com.microsoft.azure.servicebus.perf.core.ServiceBatchTest;
+import com.microsoft.azure.servicebus.perf.core.ServiceBusStressOptions;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class SendBatchTest extends ServiceBatchTest<ServiceBusStressOptions> {
+    private final List<IMessage> messages;
+
+    /**
+     * Creates an instance of Batch performance test.
+     *
+     * @param options the options configured for the test.
+     * @throws IllegalStateException if SSL context cannot be created.
+     */
+    public SendBatchTest(ServiceBusStressOptions options) {
+        super(options);
+        String messageContent = TestDataCreationHelper.generateRandomString(options.getMessagesSizeBytesToSend());
```

Comment:
The Random String can be generated only once in the constructor and reused in the for loop below.